pandas/errors/__init__.py

```diff
@@ -53,10 +53,44 @@ class EmptyDataError(ValueError):
 
 class ParserWarning(Warning):
     """
-    Warning that is raised in `pd.read_csv` whenever it is necessary
-    to change parsers (generally from 'c' to 'python') contrary to the
-    one specified by the user due to lack of support or functionality for
-    parsing particular attributes of a CSV file with the requested engine.
+    Warning raised when reading a file that doesn't use the default parser.
+
+    Thrown by `pd.read_csv` and `pd.read_table` when it is necessary to
```

Comment:
Thrown -> Raised

Reply:
Jeff, I made all the changes you requested. Thank you for the feedback.